core/libs/commonwealth/commonwealth/utils/general.py

```diff
@@ -1,5 +1,23 @@
 import os
 
+from loguru import logger
+
 
 def is_running_as_root() -> bool:
     return os.geteuid() == 0
+
+
+def device_id() -> str:
+    try:
+        with open("/sys/firmware/devicetree/base/serial-number", "r", encoding="utf-8") as f:
+            return f.read()
+    except Exception as error:
```

Comment:
This raise is weird, why we need to do it ? Can't we just fall back and try machine-id instead and have just a since raise in the end or return an optional ?

Reply:
There's no raise here.

Reply:
My fault!